app-extension/src/boot/query-cache.js

```diff
@@ -0,0 +1,62 @@
+import { addMany, findAll } from '@bildvitta/quasar-ui-asteroid/src/composables/use-cached-filters'
+import useHistory from '@bildvitta/quasar-ui-asteroid/src/composables/use-history'
+
+let isReplacingQuery = false
+
+function getQueriesToExclude (route) {
+  const { exclude = [], include = [] } = route.meta || {}
+
+  return exclude.filter(value => !include.includes(value))
+}
+
+function setRouteCache (route) {
+  if (!route.meta.useCache) return
+
+  const filteredQuery = {}
+  const { query } = route || {}
+
+  const queriesToExclude = getQueriesToExclude(route)
+
+  for (const filter in query) {
```

Comment:
Acho que seria interessante conferir se há queries para ignorar antes de realizar esse `for`. Pois se não houver, você meio que estará realizando um `for` atoa.

Reply:
Feito